### DISCUSSION TITLE
OWASP Machine Learning Security Top 10 - Draft release v0.2 - ML01:2023 Adversarial Attack

### DISCUSSION THREAD

**Discussion by giscus** [OWASP/www-project-machine-learning-security-top-10#103] [Machine Learning Security Topics]
# OWASP Machine Learning Security Top 10 - Draft release v0.2 - ML01:2023 Adversarial Attack

https://mltop10.info/ML01_2023-Adversarial_Attack.html

**Comment by shsingh** [comment]
testing comment

**Comment by shsingh** [comment]
this is another test
